content/usage/secrets/registries_global.md

```diff
@@ -47,8 +47,14 @@ Example registry credentials file:
     }
 ```
 
+{{% alert info %}}
```

Comment:
Should this note be updated since `0.8.4` has been released?  Something like

```
In versions < `0.8.4` credentials configured via global registries take precedence over credentials configured in individual repositories.
```

Reply:
Updated